var quiz = [
    

    {
        q: `<!--1. -->Deoxyribonucleic acid (DNA) is composed of all of the following elements EXCEPT:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->a base, either a purine or a pyrimidine.`, 
        `<!--b. -->a sugar, called ribose.`, 
        `<!--c. -->a phosphate.`, 
        `<!--d. -->two complementary strands.`, 
        `<!--e. -->hydrogen bonds.`],
        optionStyleType: `st-lower-alpha`,
        answer: [1],
        ansText:`Rationale: Ribose is an element of RNA, not DNA. In a rudimentary form, DNA is the fusion of three different elements: a base (either a pyrimidine or a purine), a sugar (in the case of DNA called 2-deoxyribose; for RNA called ribose), and a phosphate (which links individual nucleotides together). The repeating connections between the phosphates and the sugars provide the backbone from which the information-carrying bases protrude. In its “resting” or non-replicating form, this chain of elements forms a helix of two complementary strands; this double helix is held together by the hydrogen bonds.`, 
        state: `notAnswered`,
        userAnswered: []
    ,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`        ,
        type: `MCSS`},
    {
        q: `<!--2. -->The physical chemistry of DNA bases requires:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->uracil to form hydrogen bonds with guanine.`, 
        `<!--b. -->purine to form hydrogen bonds with another purine.`, 
        `<!--c. -->pyrimidine to form hydrogen bonds with a purine.`, 
        `<!--d. -->adenine to form hydrogen bonds with cytosine.`, 
        `<!--e. -->thymine to form hydrogen bonds with guanine.`],
        optionStyleType: `st-lower-alpha`,
        answer: [1],
        ansText:`Rationale: The double helix is held together by the formation of hydrogen bonds between the pyrimidine on one strand and the purine base on the other. Uracil is in RNA not DNA. Purines do not form bases with purines. The purine adenine (A) always forms two hydrogen bonds to the pyrimidine thymine (T), and the purine guanine (G) always forms three hydrogen bonds to the pyrimidine cytosine (C), one consequence being that A-T bonding is weaker than G-C bonding.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--3. -->Which of the following does DNA gene expression require?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->Linear DNA to be converted into linear ribonucleic acid (RNA), a process called translation`, 
        `<!--b. -->Conversion of linear RNA into a linear set of amino acids, a process called transcription`, 
        `<!--c. -->Protein synthesis exclusively within the nucleus`, 
        `<!--d. -->Mitosis`, 
        `<!--e. -->A mechanism to bridge the gap between the genetic code and protein synthesis`],
        optionStyleType: `st-lower-alpha`,
        answer: [4],
        ansText:`Rationale: The physical locations of DNA and its genetic code and protein synthesis and the manifestation of that code are separate: DNA is in the nucleus, and protein synthesis is cytoplasmic. The DNA message must be converted into mRNA by a process called transcription in the nucleus. The mRNA is transferred into the cytoplasm where the mRNA is converted into a protein by a process called translation.`, 
        state: `notAnswered`,
        userAnswered: []
    ,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`        ,
        type: `MCSS`},
    {
        q: `<!--4. -->Which of the following statements about transcriptional regulation is NOT true?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->Two general components involved in transcriptional regulation are specific sequences in the RNA and proteins that interact with those sequences.`, 
        `<!--b. -->In addition to the genetic information carried within the nucleotide sequence of DNA, it provides specific docking sites for proteins that enhance the activity of the transcriptional machinery.`, 
        `<!--c. -->Specific sequences within the promoter or enhancer region of a gene are called response elements.`, 
        `<!--d. -->DNA sequences, often referred to as consensus sequences, are found in many genes and respond in a coordinated manner to a specific signal.`, 
        `<!--e. -->It is an important mechanism to ensure coordinated gene expression.`],
        optionStyleType: `st-lower-alpha`,
        answer: [0],
        ansText:`Rationale: Specific sequences in the DNA called response elements bind to the nuclear proteins that control transcription. Importantly, this provides a mechanism to coordinate gene expression by providing similar docking sites for genes that need to be expressed at specific time points. Whereas mRNA transcription, stability, transport, and translation are all highly regulated, they are not controlled by the transcriptional machinery binding to specific mRNA sequences.`, 
        state: `notAnswered`,
        userAnswered: []
    ,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`        ,
        type: `MCSS`},
    {
        q: `<!--5. -->What is alternative splicing?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->A form of protein modification occurring after a mature polypeptide is produced`, 
        `<!--b. -->A modification to DNA during meiosis`, 
        `<!--c. -->A process of including or excluding certain exons in an mRNA transcript`, 
        `<!--d. -->A process in which novel RNA sequences are randomly inserted into a transcript`, 
        `<!--e. -->A method of RNA degradation`],
        optionStyleType: `st-lower-alpha`,
        answer: [2],
        ansText:`Rationale: A specific gene may have multiple similar (not identical) forms (isoforms). This is accomplished by having specific exons included or excluded from the final mRNA transcript. This allows one DNA sequence to produce several protein products that have different functions. It is not a random process but is very tightly regulated to ensure that the correct mRNA transcript is produced in the correct cell at the correct point in time.`, 
        state: `notAnswered`,
        userAnswered: []
    ,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`        ,
        type: `MCSS`},
    {
        q: `<!--6. -->Which of the following statements is TRUE regarding the nuclear matrix?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->It has the same protein composition in every tissue type.`, 
        `<!--b. -->It is the site of mRNA transcription.`, 
        `<!--c. -->It has the same protein composition whether a cell is proliferating or undergoing differentiation.`, 
        `<!--d. -->It provides a mechanism to trace the cell type of origin for a cancer, because the nuclear matrix is identical within tissue types.`, 
        `<!--e. -->It is a form of DNA.`],
        optionStyleType: `st-lower-alpha`,
        answer: [1],
        ansText:` `, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--7. -->What happens in the process of translation?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->The RNA message of four parts (the nucleotides A, U, C, and G) is converted into 20 amino acids by using a functional group of three adjacent nucleotides called a codon.`, 
        `<!--b. -->Each amino acid is encoded by only one codon.`, 
        `<!--c. -->Shifts in the reading frame are of no consequence in the production of the polypeptide chain because of the fidelity of template DNA.`, 
        `<!--d. -->Single-base substitutions always encode for the identical amino acid, known as a polymorphism.`, 
        `<!--e. -->Transfer of genetic information from the DNA to the RNA occurs.`],
        optionStyleType: `st-lower-alpha`,
        answer: [0],
        ansText:`Rationale: There is significant redundancy with several codons encoding for each amino acid. As a result, the RNA sequence between individuals can be different and yet still encode the same amino acid sequence (a polymorphism). However, some polymorphisms do result in amino acid changes. Shifts in reading frame are the most deleterious mutations because they result in a change in all the codons after the insertion/deletion and as a result a dramatic change in the amino acid sequence.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--8. -->Which of the following statements about ubiquitization is NOT true?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->Ubiquitization is an important regulatory mechanism of a cell, used in the efficient disposal of proteins.`, 
        `<!--b. -->A small protein called ubiquitin is linked to a protein, tagging it for destruction.`, 
        `<!--c. -->The proteosome is the site of protein-ubiquitin complex degradation.`, 
        `<!--d. -->The proteosome has a cylindrical shape.`, 
        `<!--e. -->The targeted inhibition of proteosome function appears to enhance cancer progression.`],
        optionStyleType: `st-lower-alpha`,
        answer: [4],
        ansText:`Rationale: Degradation of proteins in the cell is an active, not passive, process. Ubiquitinization is the process by which proteins are tagged for transport to the proteosome for destruction. Perturbation of this process is often found in cancer, and therefore inhibition of ubiquitinization is a promising therapy for cancer.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--9. -->Which of the following statements about oncogenes is TRUE?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->They are mutated forms of abnormal genes, known as proto-oncogenes.`, 
        `<!--b. -->They can be produced by an inactivating mutation of a proto-oncogene, resulting in the silencing of the gene.`, 
        `<!--c. -->They can be produced by gene amplification, resulting in many copies of the gene, or by chromosomal rearrangement.`, 
        `<!--d. -->They are always due to retroviruses, capable of inducing malignant transformation of normal cells.`, 
        `<!--e. -->They are endogenous cancer-fighting genes.`],
        optionStyleType: `st-lower-alpha`,
        answer: [2],
        ansText:`Rationale: There are at least three ways a proto-oncogene can be converted into an oncogene. First, a mutation can occur within the coding sequence, producing a permanently activated form of the gene. A second mechanism converting a proto-oncogene into an oncogene is through gene amplification. A third mechanism of oncogene formation is through chromosomal rearrangement.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--10. -->Which of the following statements about <i>hyper</i>methylation is TRUE?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->It is a direct change to the DNA sequence, similar to a mutation that alters the normal base-pairing.`, 
        `<!--b. -->It occurs exclusively on cytosine nucleotides in the dinucleotide sequence CG.`, 
        `<!--c. -->Somatic methylation of CpG dinucleotides in the regulatory regions of genes is very often associated with increased transcriptional activity, leading to increased expression of that gene.`, 
        `<!--d. -->In cancer, hypermethylation is associated with enhanced activity of oncogenes, and demethylation may be an effective strategy for the treatment of cancer.`, 
        `<!--e. -->It marks cells for ubiquitinization.`],
        optionStyleType: `st-lower-alpha`,
        answer: [1],
        ansText:`Rationale: Hypermethylation is a normal process by which DNA is modified by the addition of a methyl group to a cytosine nucleotide in a CpG DNA sequence. There is no change in DNA sequence. Methylation results in gene silencing, that is, decreased expression of the gene. As a result, hypermethylation in cancer is associated with decreased transcription and, therefore, reduces expression of tumor suppressor genes. Ubiquitinization decreases levels of proteins but does so by increasing the degradation of the protein. Whereas alterations in ubiquitinization occur in cancer, they are not directly related to methylation.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--11. -->Von Hippel-Lindau disease predisposes patients to:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->epididymal carcinoma`, 
        `<!--b. -->clear cell renal carcinoma`, 
        `<!--c. -->papillary renal cell carcinoma`, 
        `<!--d. -->adrenocortical carcinoma`, 
        `<!--e. -->all of the above`],
        optionStyleType: `st-lower-alpha`,
        answer: [1],
        ansText:`Rationale: VHL is a hereditary tumor syndrome that predisposes patients to clear cell renal carcinoma, retinal angiomas, pheochromocytomas, hemangiomas of the CNS, epididymal cystadenomas, and pancreatic islet cell tumors. It is not associated with epididymal, papillary renal cell, or adrenocortical carcinomas.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--12. -->Hereditary prostate cancer genes that have been proven to cause prostate carcinoma include:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->ELAC2 (elaC ribonuclease Z 2)`, 
        `<!--b. -->MSR1 (macrophage scavenger receptor 1)`, 
        `<!--c. -->RNASEL (ribonuclease L)`, 
        `<!--d. -->none of the above`, 
        `<!--e. -->all of the above`],
        optionStyleType: `st-lower-alpha`,
        answer: [3],
        ansText:`Rationale: All of the genes above have been linked to prostate cancer by both linkage studies examining families with a strong predisposition to prostate cancer and in some cases by case control studies examining polymorphisms within these genes. However, no data have conclusively demonstrated that these genes cause prostate cancer.`, 
        state: `notAnswered`,
        userAnswered: []
    ,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`,
	type: `MCSS`        ,
        type: `MCSS`},
    {
        q: `<!--13. -->Which of the following hereditary tumor syndromes are associated with genitourinary tumors?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->Von-Hippel-Lindau syndrome`, 
        `<!--b. -->Birt-Hogg-Dubé syndrome`, 
        `<!--c. -->Beckwith-Wiedemann syndrome`, 
        `<!--d. -->None of the above`, 
        `<!--e. -->All of the above`],
        optionStyleType: `st-lower-alpha`,
        answer: [4],
        ansText:`Rationale: Each of these syndromes includes increased risk of specific genitourinary malignancies among their spectra of pathologies.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--14. -->Which of the following cancer-associated chromosomal abnormalities would be most likely to be associated with inactivation of a tumor suppressor gene?`,
        q2: ``,
        q3: [],
        option: [`Inversion`, 
        `Tetraploidy`, 
        `Amplification`, 
        `Deletion`, 
        `<!--e. -->Double Minutes`],
        optionStyleType: `st-lower-alpha`,
        answer: [3],
        ansText:`Rationale: A common mechanism of tumor suppressor gene inactivation is through deletion of the gene or a chromosomal region containing the gene. The other abnormalities listed produce either no net loss of genetic material or lead to a gain of genetic material, a finding often associated with increased oncogene activity due to increased copy number of the oncogene.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--15. -->The two main points of control in the cell cycle are:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->S and G<sub>0</sub>`, 
        `<!--b. -->S and the G<sub>2</sub>M boundary`, 
        `<!--c. -->M and the G<sub>1</sub>S boundary`, 
        `<!--d. -->G<sub>1</sub>S and G<sub>2</sub>M boundaries`, 
        `<!--e. -->G<sub>2</sub> and M`],
        optionStyleType: `st-lower-alpha`,
        answer: [3],
        ansText:` `, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--16. -->The TP53 tumor suppressor gene plays a critical role in which of the following processes?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->Apoptosis`, 
        `Angiogenesis`, 
        `<!--c. -->DNA replication`, 
        `<!--d. -->Signal transduction`, 
        `<!--e. -->All of the above`],
        optionStyleType: `st-lower-alpha`,
        answer: [0],
        ansText:`Rationale: Active TP53 binds to the promoter region of TP53-responsive genes and stimulates the transcription of genes responsible for cell cycle arrest, repair of DNA damage, and apoptosis. TP53 responds to DNA damage by inducing cell cycle arrest through CDKN1A (p21cip1) and then transcriptionally activating DNA repair enzymes. If the cell cannot arrest growth and/or repair the DNA, TP53 induces apoptosis. Whereas angiogenesis, DNA replication, and signal transduction are all critical cellular processes, TP53 does not directly influence any of them.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--17. -->INK4 family members inhibit the activity of:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->cyclin D/cdk4.`, 
        `<!--b. -->cyclin E/cdk2.`, 
        `<!--c. -->cyclin A/cdk2.`, 
        `<!--d. -->cyclin B/cdc2.`, 
        `<!--e. -->all of the above.`],
        optionStyleType: `st-lower-alpha`,
        answer: [0],
        ansText:`Rationale: The INK4 family of cdk inhibitors directly inhibits the assembly of cyclin D with cdk4 and cdk6 by blocking the phosphorylation of the cyclin D-cdk4/6 complex. This phosphorylation is necessary for activation of the complex.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--18. -->Cyclin-cdk complexes primarily function at the G<sub>1</sub>S boundary by:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->dephosphorylation of RB1.`, 
        `<!--b. -->phosphorylation of MDM2.`, 
        `<!--c. -->dephosphorylation of E2F.`, 
        `<!--d. -->phosphorylation of E2F.`, 
        `<!--e. -->phosphorylation of RB1.`],
        optionStyleType: `st-lower-alpha`,
        answer: [4],
        ansText:`Rationale: Phosphorylation is the attachment of a phosphate to a protein. It alters the conformation of the protein and therefore is an excellent method of regulating gene function. Cyclin-cdk complexes phosphorylate RB1 or its family members, p107 and p130. Phosphorylated RB1 can no longer bind to members of the E2F family of transcription factors. Free E2F heterodimerizes with DP1 or DP2 and transcriptionally activates genes important in DNA replication, such as DNA polymerase-A and the cell cycle, such as E2F-1. Dephosphorylation of RB has the opposite effect. E2F regulation does not occur directly though phosphorylation or dephosphorylation. MDM2 regulates TP53 and is not directly affected by cyclin-cdk complexes.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--19. -->The regulatory proteins at the G<sub>2</sub>M checkpoint primarily respond to:`,
        q2: ``,
        q3: [],
        option: [`hypoxia.`, 
        `<!--b. -->nutrient poor environment.`, 
        `<!--c. -->DNA damage.`, 
        `<!--d. -->cytokines.`, 
        `<!--e. -->all of the above.`],
        optionStyleType: `st-lower-alpha`,
        answer: [2],
        ansText:`Rationale: Hypoxia, nutrient-poor environment, and cytokines are all signals to the cell that it is not in an environment that is conducive to cellular division. These signals influence the cell BEFORE it invests its energy in replicating the DNA, that is, it is a G1S checkpoint. If DNA damage occurs, it is critical the errors are repaired before cellular division. Therefore, DNA damage can lead to cell cycle arrest at both the G<sub>1</sub>S and G<sub>2</sub>M checkpoints.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--20. -->Nucleotide excision repair (NER) primarily protects the cell from DNA damage caused by:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->reactive oxygen species.`, 
        `<!--b. -->DNA polymerase errors.`, 
        `<!--c. -->double-stranded breaks.`, 
        `<!--d. -->ultraviolet light.`, 
        `<!--e. -->all of the above.`],
        optionStyleType: `st-lower-alpha`,
        answer: [3],
        ansText:`Rationale: Nucleotide excision repair (NER) is a major defense against DNA damage caused by ultraviolet radiation and chemical exposure. NER acts on a wide range of alterations that result in large local distortions in DNA by recognizing distortions in the DNA helix, excising the damaged DNA, and replacing it with the correct sequence. Base excision repair (BER) is the primary mechanism for repairing damage caused by reactive oxygen species. MMR is the primary mechanism for polymerase errors. Double-strand break repair is the primary mechanism for repairing double-strand breaks.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--21. -->Which of the following repair pathways is responsible for repairing double-strand DNA breaks?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->MSH2/MSH6`, 
        `<!--b. -->homologous recombination (HR)`, 
        `NER`, 
        `<!--d. -->mismatch repair (MMR)`, 
        `<!--e. -->none of the above`],
        optionStyleType: `st-lower-alpha`,
        answer: [1],
        ansText:`Rationale: HR is one of two mechanisms for repairing DNA double-strand breaks, the other being non-homologous end joining (NHEJ). In HR, the normal undamaged sister chromatid is used as a template to repair the damaged segment of DNA.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--22. -->Which of the following genes has been linked to double-strand break repair?`,
        q2: ``,
        q3: [],
        option: [`TP53`, 
        `<!--b. -->VHL (von Hippel-Lindau tumor suppressor)`, 
        `<!--c. -->BRCA1 (breast cancer type 1 susceptibility protein)`, 
        `<!--d. -->RB1 (retinoblastoma 1)`, 
        `<!--e. -->PTEN (phosphatase and tensin homolog)`],
        optionStyleType: `st-lower-alpha`,
        answer: [2],
        ansText:`Rationale:  BRCA1 is associated with familial breast and ovarian cancer. It is believed that the breast cancer susceptibility gene BRCA1 (as well as BRCA2) plays an important role in HR as well as sensing DNA damage. Both BRCA1 and BRCA2 are part of an enzymatic complex with RAD51- and BRCA1-associated RING domain 1 (BARD1). This complex is recruited by proliferating cell nuclear antigen (PCNA) to regions that have undergone DNA damage to repair DNA breaks.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--23. -->Procaspases are activated by which of the following?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->Phosphorylation`, 
        `Ubiquitination`, 
        `Dimerization`, 
        `<!--d. -->Mitochondrial import`, 
        `<!--e. -->Proteolytic cleavage`],
        optionStyleType: `st-lower-alpha`,
        answer: [4],
        ansText:`Rationale: Procaspases are the larger, inactive precursor forms of the caspase proteins. Specific proteolytic cleavage is required for their activation. The caspases themselves are proteases and, once activated, proceed to cleave and activate other caspases, thus facilitating a proteolytic cascade that serves to amplify the initial apoptotic signal.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--24. -->Ligand-dependent apoptosis is an attractive therapeutic target because activation is:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->independent of TP53.`, 
        `<!--b. -->dependent on TP53.`, 
        `<!--c. -->independent of caspases`, 
        `<!--d. -->dependent on caspases.`, 
        `<!--e. -->dependent on RB1.`],
        optionStyleType: `st-lower-alpha`,
        answer: [0],
        ansText:`Rationale: The identification of ligand-dependent apoptosis receptors may have a profound impact on therapy. Most cancer therapies (e.g., chemotherapy and external beam radiotherapy) depend on TP53 to induce apoptosis in the cancer cell. Because TP53 is mutated in more than half of malignancies, TP53-independent pathways for apoptosis are of great clinical interest. Because ligand-dependent apoptosis is independent of TP53, these receptors and ligands are attractive and novel treatment targets.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--25. -->The TP53-induced apoptosis is mediated through:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->APAF-1/caspase 9.`, 
        `<!--b. -->CD95 receptor.`, 
        `TRAIL.`, 
        `<!--d. -->Bcl-2.`, 
        `<!--e. -->CDKN1A (p21<sup>cip</sup>).`],
        optionStyleType: `st-lower-alpha`,
        answer: [0],
        ansText:`Rationale: The TP53-induced apoptosis is dependent on the APAF-1/caspase 9 activation pathway.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--26. -->How do proapoptotic bcl-2 family members function? By:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->digesting the mitochondria.`, 
        `<!--b. -->increasing the cellular membrane permeability.`, 
        `<!--c. -->increasing the mitochondrial membrane permeability.`, 
        `<!--d. -->directly activating executioner caspases.`, 
        `<!--e. -->increasing nuclear membrane permeability.`],
        optionStyleType: `st-lower-alpha`,
        answer: [2],
        ansText:`Rationale: Although each proapoptotic bcl-2 family member responds to different stimuli, the principal mechanism by which these family members induce cell death is by increasing mitochondrial membrane permeability.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--27. -->The enzyme telomerase immortalizes cells by:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->protecting the cells from DNA damage.`, 
        `<!--b. -->stabilizing TP53.`, 
        `<!--c. -->allowing the cell to grow in a nutrient poor environment.`, 
        `<!--d. -->inhibiting apoptosis.`, 
        `<!--e. -->maintaining chromosomal length.`],
        optionStyleType: `st-lower-alpha`,
        answer: [4],
        ansText:`Rationale: Telomerase immortalizes cells by maintaining the ends of the chromosomes, or telomeres, which normally shorten with each cell division.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--28. -->Telomere loss can lead to all of the following EXCEPT:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->irreversible cell cycle exit termed senescence.`, 
        `<!--b. -->apoptosis.`, 
        `<!--c. -->DNA hypomethylation.`, 
        `<!--d. -->chromosomal instability.`, 
        `<!--e. -->increased tumor initiation.`],
        optionStyleType: `st-lower-alpha`,
        answer: [2],
        ansText:`Rationale: Normal cells closely monitor their telomere lengths and, should they fall below a critical threshold length, will initiate either cell senescence or apoptosis. If key players in these responses (e.g., TP53) are mutated, then chromosomal instability may result contributing to cancer initiation. To date, there is not a strong connection known for telomere loss and loss of DNA methylation.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--29. -->Which of the following chromosomal rearrangements is not typically associated with a genitourinary malignancy?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->Fusion of BCR to ABL via chromosome translocation`, 
        `<!--b. -->Fusion of TMPRSS2 to ERG via intra-chromosomal deletion`, 
        `<!--c. -->Fusion of MiTF/TFE gene family members via chromosome translocation`, 
        `<!--d. -->Isochromosome 8q`, 
        `<!--e. -->Loss of chromosome 9`],
        optionStyleType: `st-lower-alpha`,
        answer: [0],
        ansText:`Rationale: This particular chromosomal abnormality is typically found in chronic myelogenous leukemia patients, not in solid tumors. TMPRSS2-ERG gene fusions are found in ∼50% of prostate cancer cases, MiTF/TFE gene family translocations have been associated with a subset of renal cell carcinomas, isochromosome 8q is found in a subset of prostate cancers and is often associated with the loss of 8p, loss of chromosome 9 is observed in a subset of urothelial cancers and enumeration of this chromosome by FISH is one of the components of a molecular test used for detecting bladder cancer.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--30. -->An isochromosome of 12p has been identified in which genitourinary carcinoma?`,
        q2: ``,
        q3: [],
        option: [`Testis`, 
        `Prostate`, 
        `Renal`, 
        `Bladder`, 
        `Penile`],
        optionStyleType: `st-lower-alpha`,
        answer: [0],
        ansText:`Rationale: The urologic malignancy most closely linked to a karyotypic abnormality is a testis tumor. A 12p isochromosome was first identified in a testis tumor in the 1980s, and experimentally this cytogenetic hallmark of testicular tumors has diagnostic and prognostic value.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--31. -->Which of the following has been most consistently implicated in prostate cancer risk from GWAS studies?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->The 8q24 chromosomal region`, 
        `MSR1`, 
        `RNASEL`, 
        `<!--d. -->TMPRSS2-ERG fusions`, 
        `<!--e. -->None of the above`],
        optionStyleType: `st-lower-alpha`,
        answer: [0],
        ansText:`Rationale: Systematic review of GWAS studies in prostate cancer indicate that the 8q24 region continues to be the most implicated in prostate cancer risk, and among different racial cohorts.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--32. -->What type of genetic alteration in cancer could be identified using next-generation RNA sequencing (RNA-seq)?`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->MicroRNAs associated with cancer`, 
        `<!--b. -->Non-coding RNAs associated with cancer`, 
        `<!--c. -->Gene fusions`, 
        `<!--d. -->mRNA overexpression in cancer`, 
        `<!--e. -->All of the above`],
        optionStyleType: `st-lower-alpha`,
        answer: [4],
        ansText:`Rationale: RNA-seq technologies can unbiasedly sequence all RNA species in a sample, with an analysis that is not limited to “annotated” sequences.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--33. -->An advantage of DNA sequencing-based technologies over standard clinical microbial culture for pathogen identification in body sites such as the urinary tract is:`,
        q2: ``,
        q3: [],
        option: [`<!--a. -->sequencing techniques can be used to define bacterial growth conditions`, 
        `<!--b. -->sequencing technologies can identify the presence of organisms that cannot be grown in culture`, 
        `<!--c. -->microbial culture cannot assist with the classification of bacterial species`, 
        `<!--d. -->microbial culture cannot accurately identify pathogens`, 
        `<!--e. -->none of the above`],
        optionStyleType: `st-lower-alpha`,
        answer: [1],
        ansText:`Rationale: This serves as an advantage over standard microbial culture because many microorganisms that are present as members of the human microbiome are not readily culturable with standard clinical microbiology protocols.`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    {
        q: `<!--34. -->Which of the following medications has been shown to improve overall survival with fewer severe side effects among patients with clear cell renal cell carcinoma who received previous treatment?`,
        q2: ``,
        q3: [],
        option: [`Atezolizumab`, 
        `Pembrolizumab`, 
        `Everolimus`, 
        `Nivolumab`, 
        `<!--e. -->None of the above`],
        optionStyleType: `st-lower-alpha`,
        answer: [3],
        ansText:`Rationale: A randomized, open-label, phase 3 study comparing nivolumab with everolimus in patients with renal-cell carcinoma who received previous treatment demonstrated improved overall survival and fewer severe adverse events with nivolumab therapy (Motzer et al., 2015).`, 
        state: `notAnswered`,
        userAnswered: []
            ,
        type: `MCSS`},
    


];
